### DISCUSSION TITLE
in rootless mode, /tmp/podman-run-{myuid}/podman/podman.sock was not created

### DISCUSSION THREAD

**Discussion by yuki00200007** [containers/podman#23078] [Q&A] [answered]
Hi, I'm using podman(podman version 4.9.4-rhel) in rootless mode.
When I input `$ podman info`, I found that the remoteSocket file does not exist.
Is it normal?
OS: Rocky Linux 9.3

A part of the result of `$ podman info`
```
  remoteSocket:
    exists: false
    path: /tmp/podman-run-1100/podman/podman.sock
```

Thank you!

**Comment by rhatdan** [comment, marked as answer]
Yes on a Linux platform podman works locally and does not use the socket.  If you want to use podman --remote on a local system you need to activate the podman.socket systemd unit file.

**Reply by rhatdan** [reply]
podman is different then Docker in that by default it does not require a demaon or service running, it just fork/exec to launch the container.  Docker talks to a daemon, usually running as root, and the daemon launches the container.
